jerry-core/debugger/debugger-tcp.c

```diff
@@ -84,8 +92,8 @@ jerry_debugger_tcp_send (jerry_debugger_transport_header_t *header_p, /**< tcp i
         continue;
       }
 
-      jerry_debugger_tcp_log_error ();
       jerry_debugger_transport_close ();
+      jerry_debugger_tcp_log_error ();
```

Comment:
Isn't socket `close` will overwrite `errno`?

Reply:
I get the point: don't send the message through the network. Then save `errno` into a local variable, and call log_error with that value.